uber/site_sections/preregistration.py

```diff
@@ -339,13 +339,12 @@ def process_group_payment(self, session, payment_id, stripeToken):
         else:
             group.amount_paid += charge.dollar_amount
 
-            for attendee in charge.attendees:
+            for attendee in group.attendees:
```

Comment:
@EliAndrewC - I would love an explanation for _why_ this change fixes #1584, because frankly, I'm a bit lost.